core-feature-pack/src/main/resources/content/bin/common.ps1

```diff
@@ -34,7 +34,7 @@ $global:SECMGR = Get-Env-Boolean SECMGR $false
 $global:DEBUG_MODE=Get-Env DEBUG $false
 $global:DEBUG_PORT=Get-Env DEBUG_PORT 8787
 $global:RUN_IN_BACKGROUND=$false
-$global:GC_LOG=Get-Env GC_LOG $false
+$GC_LOG=Get-Env GC_LOG $false
```

Comment:
@psakar for downstream, only change needed would be to replace this line with
`$GC_LOG=Get-Env GC_LOG $false`